shell/platform/common/cpp/client_wrapper/include/flutter/texture_registrar.h

```diff
@@ -42,10 +42,50 @@ class PixelBufferTexture {
   const CopyBufferCallback copy_buffer_callback_;
 };
 
+// A gpu buffer texture.
+class GpuBufferTexture {
+ public:
+  // A callback used for retrieving gpu buffers.
+  typedef std::function<const FlutterDesktopGpuBuffer*(size_t width,
+                                                       size_t height)>
+      ObtainGpuBufferCallback;
+
+  typedef std::function<void(void* buffer)> DestructionGpuBufferCallback;
+
+  // Creates a gpu buffer texture that uses the provided |copy_buffer_cb| to
+  // retrieve the buffer.
+  // As the callback is usually invoked from the render thread, the callee must
+  // take care of proper synchronization. It also needs to be ensured that the
+  // returned buffer isn't released prior to unregistering this texture.
+  GpuBufferTexture(ObtainGpuBufferCallback obtain_buffer_callback,
```

Comment:
You forgot to update the documentation.

```suggestion
  // Creates a gpu buffer texture that uses the provided
  // |obtain_buffer_callback| to retrieve the buffer.
  // As the callback is usually invoked from the render thread, the callee must
  // take care of proper synchronization. It also needs to be ensured that the
  // returned buffer isn't released prior to unregistering this texture.
  GpuBufferTexture(ObtainGpuBufferCallback obtain_buffer_callback,
```

Reply:
Fixed